plugins/BEdita/Core/src/Model/Table/UsersTable.php

```diff
@@ -167,6 +168,20 @@ public function buildRules(RulesChecker $rules): RulesChecker
         return $rules;
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * @codeCoverageIgnore
+     */
+    protected function _initializeSchema(TableSchema $schema): TableSchema
+    {
+        if ($schema->getColumnType('user_preferences') !== 'json') {
+            $schema->setColumnType('user_preferences', 'json');
+        }
+
+        return $schema;
```

Comment:
Would there be any undesirable side effect in setting `json` right away?

```suggestion
        return $schema
            ->setColumnType('user_preferences', 'json');
```